README.md

```diff
@@ -102,39 +102,61 @@ To see the Python instrumentation in action with sample applications, see our
-| OTEL_EXPORTER_JAEGER_ENDPOINT | endpoint                             | `http://localhost:9080/v1/trace`     | The jaeger endpoint to connect to. Currently only HTTP is supported.   |
+| OTEL_TRACES_EXPORTER | exporter_factories | `otlp`     | The exporter(s) that should be used to export tracing data. |
+| OTEL_EXPORTER_OTLP_ENDPOINT |  | `http://localhost:4317`     | The OTLP gRPC endpoint to connect to. Used when `OTEL_TRACES_EXPORTER` is set to `otlp` |
+| OTEL_EXPORTER_JAEGER_ENDPOINT |  | `http://localhost:9080/v1/trace`     | The jaeger thrift endpoint to connect to. Used when `OTEL_TRACES_EXPORTER` is set to `jaeger-thrift-splunk` |
 | SPLUNK_ACCESS_TOKEN          | access_token |      | The optional organization access token for trace submission requests.  |
 | SPLUNK_MAX_ATTR_LENGTH       | max_attr_length | 1200            | Maximum length of string attribute value in characters. Longer values are truncated.                                                                                                                                                                                                                                                                                                                      |
 | SPLUNK_TRACE_RESPONSE_HEADER_ENABLED | trace_response_header_enabled | True | Enables adding server trace information to HTTP response headers. |
 | OTEL_RESOURCE_ATTRIBUTES      |            | unset          | Comma-separated list of resource attributes added to every reported span. <details><summary>Example</summary>`service.name=my-python-service,service.version=3.1,deployment.environment=production`</details>
 | OTEL_PROPAGATORS              |            | tracecontext,baggage   | Comma-separated list of propagator names to be used. See[Configuring Propagators](#configuring-propagators) for more details.
-| OTEL_TRACE_ENABLED            |            | `true`         | Globally enables tracer creation and auto-instrumentation.                                                                                                                                                                                                                                                                                                                                                |
+| OTEL_TRACE_ENABLED            |            | `true`         | Globally enables tracer creation and auto-instrumentation. |
 
 ## Advanced Getting Started
 
-### Alternative: List requirements instead of installing them
+### Using a different exporter
 
-The `splk-py-trace-bootstrap` command can optionally print out the list of
-packages it would install if you chose. In order to do so, pass
-`-a=requirements` CLI argument to it. For example,
+The `splunk-opentelemetry` Python package does not install any exporters by default. You can install it with the OTLP or Jaeger thrift exporter by
```

Comment:
nit: Inconsistent capitalization for Jaeger and the Thrift exporter. I suggest using capital letters at least for "Jaeger" and "Thrift"